### DISCUSSION TITLE
[1회] 과제(~02/12)

### DISCUSSION THREAD

**Discussion by s9hn** [like-comment-subscribe-and-Algorithm/sehunTV#2] [General]
1. 알고리즘 2문제풀기
   - 완전탐색 : 부분집합의 합
   - 그리디 : 회의실 배정
2. 공동계좌 들어오기
3. 오가니 초대받고 레포파기

**Comment by KWY0218** [comment]
@s9hn 
뒷북 죄송함다~
- 챕터 지정 문제(1) + 챕터 자유 문제(2) 맞을까요??
    - 그럼 매주 3챕터 씩 나가면 3 + 3 + 3이 될텐데 나머지 하나는 챕터 중 추가로 풀고 싶은 거 하나로 하면 될까요??
    - 또한, 이번주는 완전 탐색, 그리디 2챕터이기 때문에 6문제일까요??

**Reply by s9hn** [reply]
@KWY0218 

챕터 지정 문제(1)만 지키는걸로하고 자유문제는 자유롭게 선택해서 푸시죠! 해당 챕터 자신있으면  다른챕터에서 7문제 푸셔도 상관없습니다!
이번주는 첫 주차이니 지정문제 2문제만 필수고 나머지는 자유입니다.
